src/components/network-modification-table/renderers/switch-cell.tsx

```
@@ -0,0 +1,80 @@
+        >
+            <span>
+                <Switch
+                    size="small"
+                    disabled={isLoading || isDisabled}
+                    checked={modificationActivated}
+                    onChange={toggleModificationActive}
+                />
+            </span>
+        </Tooltip>
+    );
+};
```

ESLint

[error] Function component is not a function declaration

(react/function-component-definition)